global.json

```diff
@@ -3,7 +3,7 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "7.0.304",
+    "dotnet": "7.0.306",
```

Comment:
Puting a comment here to block possible auto-merge.
All is good - I just do not know if we can merge to 17.7 without any tag/tell/approval process (it's just an infra stuff, so probably not a problem - but just for sure).
Taging @rainersigwald - is this considered servicing? Or can we generaly just go ahead with infra-only changes?

Reply:
Required infra for servicing is generally ok, the approval for the subsequent servicing bug includes "and the changes you are required to make to get it to build and ship" unless they're huge--and bumping to a new released SDK is not a problem. I think this is fine.